src/Router.ts

```diff
@@ -201,13 +250,15 @@ export class Router<C extends Context> extends Evented {
 	private _history?: History;
 	private _routes: Route<Context, Parameters>[];
 	private _started?: boolean;
+	private _outletStack: Map<string, any> = new Map<string, OutletContext>();
+	private _outletRouteMap: Map<string, Route<any, any>> = new Map<string, Route<any, any>>();
```

Comment:
it's a map of outlet identifer, i.e. `string` to the `Route` instance that was created for it.